rootfs/bin/create-snapshot

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
```

Comment:
should `set -o pipefail` https://bclary.com/blog/2006/07/20/pipefail-testing-pipeline-exit-codes/